docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md

```diff
@@ -41,45 +41,52 @@ Complete the steps in one of the following sections to set up an add-in catalog
 
 9. To add an add-in to the Office Add-ins add-in catalog, choose **Apps for Office**.
 
-### To set up an add-in catalog on Office 365
+### To set up an app catalog on Office 365
 
-1. On the Office 365 admin center page, choose  **Admin**, and then choose  **SharePoint**.
+Even though SharePoint names the catalog an "app" catalog, you can register Office Add-ins in the app catalog.
 
-2. In the left task pane, choose  **add-ins**.
+1. Go to the Microsoft 365 admin center. For information on how to find the admin center, see [About the Microsoft 365 admin center](https://docs.microsoft.com/office365/admin/admin-overview/about-the-admin-center)

```

Comment:
missing a period at the end of the second sentence (after the link).

Reply:
fixed